scripts/make_coadd_dataframes.py

```diff
@@ -4,22 +4,53 @@
 import sys
 import re
 import logging
+import numpy as np
 import pandas as pd
 import lsst.daf.persistence as dp
+import lsst.pex.exceptions as pexExcept
 
 logging.basicConfig(format="%(message)s", stream=sys.stdout)
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 #logger.setLevel(logging.DEBUG)
 
 def make_dataframe(catalog):
+    """
+    Create a data frame from a coadd catalog using the coadd catalog
+    schema.
+    """
     df = pd.DataFrame()
     for col in catalog.getSchema():
         name = col.field.getName()
         df[name] = catalog.get(name)
     return df
 
+def add_calibrated_mags(df, catalog, coadd):
+    """
+    Add columns for calibrated magnitudes for psfFlux and cmodelFlux.
+    """
+    modelFlux = catalog.getModelFlux()
+    psfFlux = catalog.getPsfFlux()
+    coadd_calib = coadd.getCalib()
+    modelMags = []
+    psfMags = []
+    for mflux, pflux in zip(modelFlux, psfFlux):
+        try:
+            modelMags.append(coadd_calib.getMagnitude(mflux))
+        except pexExcept.DomainError:
+            modelMags.append(np.nan)
+        try:
+            psfMags.append(coadd_calib.getMagnitude(pflux))
+        except pexExcept.DomainError:
+            psfMags.append(np.nan)
+    df['cmodelMag'] = np.array(modelMags)
```

Comment:
@cwwalter Can you please have a look at this and confirm that it is adding the columns you want with the desired names?   If it seems ok, I'll clean up the code a bit and request a merge.  I'd like to make the hdf files this week so that we can proceed with archiving the DC1 repos.